examples/bigquery_dataset/main.tf

```diff
@@ -47,3 +47,8 @@ resource "google_bigquery_dataset" "bigquery_dataset_one" {
 resource "random_id" "test" {
   byte_length = 4
 }
+
+output "bigquery_dataset_one" {
```

Comment:
I didn't grok the reason for adding these outputs

Reply:
Lint tests are failing of there is no output thats why I had to add outputs to the examples which were missing outputs. It is strange that these tests are failing in cloudbuild. When I run tests locally they dont show as an issue.